src/order/atoms.lean

```diff
@@ -678,18 +678,13 @@ lemma is_simple_order [bounded_order α] [bounded_order β] [h : is_simple_order
   is_simple_order α :=
 f.is_simple_order_iff.mpr h
 
-lemma is_atomic_iff [order_bot α] [order_bot β] (f : α ≃o β) : is_atomic α ↔ is_atomic β :=
-begin
-  suffices : (∀ b : α, b = ⊥ ∨ ∃ (a : α), is_atom a ∧ a ≤ b) ↔
-    (∀ b : β, b = ⊥ ∨ ∃ (a : β), is_atom a ∧ a ≤ b),
-  from ⟨λ ⟨p⟩, ⟨this.mp p⟩, λ ⟨p⟩, ⟨this.mpr p⟩⟩,
-  simp only [f.surjective.forall, f.surjective.exists, ← map_bot f, f.injective.eq_iff,
-    f.le_iff_le, f.is_atom_iff]
-end
+protected lemma is_atomic_iff [order_bot α] [order_bot β] (f : α ≃o β) :
+  is_atomic α ↔ is_atomic β :=
+by simp only [is_atomic_iff, f.surjective.forall, f.surjective.exists, ← map_bot f, f.eq_iff_eq,
+  f.le_iff_le, f.is_atom_iff]
 
 lemma is_coatomic_iff [order_top α] [order_top β] (f : α ≃o β) : is_coatomic α ↔ is_coatomic β :=
```

Comment:
This one should also be protected right? Looks good otherwise, thanks!

Reply:
```suggestion
protected lemma is_coatomic_iff [order_top α] [order_top β] (f : α ≃o β) :
  is_coatomic α ↔ is_coatomic β :=
```